Script/char.dat

```diff
@@ -3031,7 +3532,68 @@ femaleslave
     {
       2,
       "\"Never say no to Petrus. Ever!\"",
-      "\"And I thought Richel Decos was bad with his whips...\"";
+      "\"And I thought Richel Decos was bad with his whips...\"";
```

Comment:
I feel this character should be fleshed out more, and maybe have a different backstory. She has fewer lines than the zombie prisoner or the old lobotomised prisoner, and the implication that refusing the high priest's sexual advances lands one in prison with a torturer doesn't really add anything positive to the game in terms of character development, narrative, or comedy.

Perhaps she discovered that the toy frog really does need to fanned?